`npm install` fails on ./frontend

```bash
tom:~/go/src/github.com/ubclaunchpad/pinpoint/frontend (master)$ npm install
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: no such file or directory, rename '/Users/tom/go/src/github.com/ubclaunchpad/pinpoint/frontend/node_modules/.staging/fsevents-55d368d7/node_modules/set-blocking' -> '/Users/tom/go/src/github.com/ubclaunchpad/pinpoint/frontend/node_modules/.staging/set-blocking-0482a2c9'

npm ERR! path /Users/tom/go/src/github.com/ubclaunchpad/pinpoint/frontend/node_modules/.staging/pinpoint-client-4678a6ae/node_modules/@sinonjs/samsam
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/Users/tom/go/src/github.com/ubclaunchpad/pinpoint/frontend/node_modules/.staging/pinpoint-client-4678a6ae/node_modules/@sinonjs/samsam' -> '/Users/tom/go/src/github.com/ubclaunchpad/pinpoint/frontend/node_modules/.staging/@sinonjs/samsam-6beb80d9'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tom/.npm/_logs/2019-01-09T04_38_29_910Z-debug.log
```

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -287,7 +287,6 @@
     "github.com/go-chi/cors",
     "github.com/go-chi/render",
     "github.com/golang/protobuf/proto",
-    "github.com/golang/protobuf/ptypes/any",
     "github.com/grpc-ecosystem/go-grpc-middleware",
     "github.com/grpc-ecosystem/go-grpc-middleware/logging/zap",
     "github.com/grpc-ecosystem/go-grpc-middleware/tags",

--- a/Makefile
+++ b/Makefile
@@ -18,8 +18,9 @@ deps:
 	go get -u github.com/maxbrunsfeld/counterfeiter
 	go get -u github.com/vburenin/ifacemaker
 	dep ensure
-	( cd frontend ; npm install )
 	( cd client ; npm install )
+	( cd frontend ; npm run preinstall )
+	( cd frontend ; npm install )
 
 # Execute tests
 .PHONY: test

--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,6 @@ deps:
 	go get -u github.com/vburenin/ifacemaker
 	dep ensure
 	( cd client ; npm install )
-	( cd frontend ; npm run preinstall )
 	( cd frontend ; npm install )
 
 # Execute tests

--- a/core/database/mocks/database.i.go
+++ b/core/database/mocks/database.i.go
@@ -2,10 +2,10 @@
 package mocks
 
 import (
-	sync "sync"
+	"sync"
 
-	database "github.com/ubclaunchpad/pinpoint/core/database"
-	model "github.com/ubclaunchpad/pinpoint/core/model"
+	"github.com/ubclaunchpad/pinpoint/core/database"
+	"github.com/ubclaunchpad/pinpoint/core/model"
 )
 
 type FakeDBClient struct {

--- a/frontend/package.json
+++ b/frontend/package.json
@@ -53,6 +53,7 @@
   "scripts": {
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
+    "preinstall": "npm install ../client/",
     "test": "node scripts/test.js --env=jsdom",
     "lint": "eslint .",
     "sass-lint": "sass-lint -v"

--- a/protobuf/fakes/pinpoint.pb.go
+++ b/protobuf/fakes/pinpoint.pb.go
@@ -2,13 +2,13 @@
 package fakes
 
 import (
-	context "context"
-	sync "sync"
+	"context"
+	"sync"
 
 	pinpoint "github.com/ubclaunchpad/pinpoint/protobuf"
-	request "github.com/ubclaunchpad/pinpoint/protobuf/request"
-	response "github.com/ubclaunchpad/pinpoint/protobuf/response"
-	grpc "google.golang.org/grpc"
+	"github.com/ubclaunchpad/pinpoint/protobuf/request"
+	"github.com/ubclaunchpad/pinpoint/protobuf/response"
+	"google.golang.org/grpc"
 )
 
 type FakeCoreClient struct {

